pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -9,8 +9,9 @@ import (
 )
 
 const (
-	clientSecretFilePath    = "/var/oauth-config/clientSecret"
-	oauthEndpointCAFilePath = "/var/router-ca/ca-bundle.crt"
+	clientSecretFilePath           = "/var/oauth-config/clientSecret"
+	oauthEndpointCAFilePath        = "/var/router-ca/ca-bundle.crt"
```

Comment:
Lets fix the name here as well, `routerCAFIlePath` vs `oauthEndpointCAFilePath`.  
`oauthEndpointCAFilePath` and `defaultOAuthEndpointCAFilePath` sound like the same thing.

Reply:
yeah, probably it will be better to give the true names with an additional comment